src/Table/Table.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import { Typography } from '../Typography';
+
+const { Cell, Row, Head, Body, HeaderCell } = Table;
+
+const StyledStory = styled('div')`
+    padding: 2rem;
+    color: text.base;
+    font-family: base;
+`;
+
+const createData = (name: string, color: string, rating: string) => ({
+    name,
+    color,
+    rating,
+});
+const data = [
+    createData('Pear', 'Green', '4/5'),
+    createData('Apple', 'Red', '2/5'),
+    createData('Banana', 'Yellow', '4/5'),
+    createData('Grapefruit', 'Orange', '3.5/5'),
+    createData('Apricot', 'Orange', '1.5/5'),
+];
+
+storiesOf('Components/Table', module).add('Default', () => (
+    <ThemeProvider theme={RootTheme}>
+        <StyledStory>
+            <Typography
+                tag="h3"
+                scale={24}
+                weight={500}
+                mb="0.5rem"
+                color="text.light"
+            >
+                Fruit Table
+            </Typography>
+            <Table
+                borderRadius="modal"
+                width="33.75rem"
+                size={select<'md' | 'sm'>('size', ['md', 'sm'], 'md')}
+            >
+                <Head>
+                    <Row>
+                        <HeaderCell scope="col" />
+                        <HeaderCell scope="col">
+                            <Typography scale={28} weight={600} tag="p">
+                                $4
+                            </Typography>
+                            <Typography
+                                scale={14}
+                                color="text.light"
+                                weight="normal"
+                            >
+                                (30 Day Qty)
+                            </Typography>
+                        </HeaderCell>
+                        <HeaderCell scope="col">
+                            <Typography scale={28} weight={600} tag="p">
+                                $10
+                            </Typography>
+                            <Typography
+                                scale={14}
+                                color="text.light"
+                                weight="normal"
+                            >
+                                (90 Day Qty)
+                            </Typography>
+                        </HeaderCell>
+                    </Row>
+                </Head>
+                <Body>
+                    {data.map(({ name, color, rating }) => (
+                        <Row>
+                            <HeaderCell
+                                scope="row"
+                                align="left"
+                                color="#91609c"
+                                fontWeight={600}
+                            >
+                                {name}
+                            </HeaderCell>
+                            <Cell
+                                align="center"
+                                width={{ xs: '100px', md: '1px' }}
```

Comment:
I think it's good to demonstrate what sets this `Table` apart from using a standard `table`. Perhaps set width's here to something really noticeable, because `1px` doesn't actually do anything since table cells can't totally collapse, and `xs` is difficult to see in Storybook right off the bat.

Reply:
The real benefit of this implementation of `table` is all the xstyled system props you baked in. I'd consider making more knobs to show off just how configurable this is.